client/js/components/statement/assessmentTable/FilterModalSelectItem.vue

```diff
@@ -9,7 +9,7 @@
 
 <template>
   <div
-    v-if="false === hidden"
+    v-if="false === hidden && (dplan.projectName === 'Planfeststellungsverfahren' && filterItem.attributes.label !== 'Datei')"
```

Comment:
I am honestly not sure if this ain't too much of buisness logic within this component. However I have no better idea how this could be solved instead, since this should only be valid for the planfestsh project. Having an template override instead appears to be kind of an overkill to me but maybe this is rather the way to go?